javascript/organizing_your_javascript_code/classes.md

```diff
@@ -10,19 +10,20 @@ After this lesson and completing the assignments, you will be able to:
 
 - Describe the pros and cons of using classes in JavaScript.
 - Briefly discuss how JavaScript's object creation differs from a language like Java or Ruby.
-- Explain the differences between using a class to define a constructor and other prototype methods.
+- Explain the differences between an object constructor and a class.
 - Explain what "getters" & "setters" are.
 - Understand what computed names and class fields are.
 - Describe function binding.
-- Be able to use inheritance with classes.
-- Briefly talk about the conflict in JS with functional programming and classes.
+- Use inheritance with classes.
+- Explain why favoring Composition over Inheritance is suggested.
```

Comment:
I think this line is fine in the knowledge checks, but could be rephrased slightly here to read a bit better / differentiate it. I would also remove the capitalization on composition / inheritance.
```suggestion
Understand why composition is generally preferred to inheritance
```
:point_up: just an idea, feel free to improve / tweak or come up with your own